src/Services/ConfigService.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types=1);
+
+namespace StrictPhp\HttpClients\Services;
+
+use StrictPhp\HttpClients\Contracts\ConfigContract;
+use StrictPhp\HttpClients\Exceptions\InvalidStateException;
+
+final class ConfigService
```

Comment:
service by měla být stateless, máme 2 možnosti:

udělat entitu která to bude držet a předávat to metody add / ...

a nebo udělat ConfigManager (manager beru jako holder).

Co myslíš? napadá tě 3 možnost?